base/applications/rapps/lang/fr-FR.rc

```diff
@@ -24,7 +24,7 @@ BEGIN
         MENUITEM "&Désinstaller\tCtrl+Suppr", ID_UNINSTALL
         MENUITEM "&Modifier", ID_MODIFY
         MENUITEM SEPARATOR
-        MENUITEM "&Supprimer du registre", ID_REGREMOVE
+        MENUITEM "Supprim&er du registre", ID_REGREMOVE
```

Comment:
This isn't a natural change in french (This could also be said for some other lang changes in the PR)

Reply:
What do you mean with isn't a natural change. Do you want the french to keep its &S ? I wouldn't mind reverting that if you prefered that.

Reply:
Ok, french &S extrawurst applied here. No problem for me.